src/main/java/at/hannibal2/skyhanni/utils/ComputerEnvDebug.kt

```diff
@@ -216,6 +223,31 @@ object ComputerEnvDebug {
 
     private fun getUptime() = ManagementFactory.getRuntimeMXBean().uptime.milliseconds
 
+    //#if MC == 1.8.9
+    private fun performanceMods(event: DebugDataCollectEvent) {
+        if (PlatformUtils.isDevEnvironment) return
+        val hasOptifine = FMLClientHandler.instance().hasOptifine()
+        val hasPatcher = Loader.isModLoaded("patcher")
+        event.title("Optifine")
+        if (hasOptifine) {
+            event.addIrrelevant("Optifine is installed")
+        } else {
+            event.addData("Optifine is not installed")
+        }
+        if (hasPatcher) {
+            event.addIrrelevant("Patcher is installed")
+        } else {
+            event.addData("Patcher is not installed")
```

Comment:
this would still have the title "optifine".
maybe change the title to "performance mods"?